### ISSUE TITLE
Bloom is sparking/blinking too much when TAA is disabled

### ISSUE THREAD

**Issue by ksuprynowicz** [overte-org/overte#1231]
Would it be possible to always use TAA for bloom calculation, but output image without TAA to the screen?

**Comment by HifiExperiments** [comment]
not exactly, really we need to compare the TAA code path with the other modes and see where we are doing an extra clamp or handling extreme values.  I think probably we are dividing by 0 somewhere and the TAA path just covers it up
